src/olympia/addons/models.py

```diff
@@ -664,18 +664,20 @@ def valid_file_statuses(self):
             return [amo.STATUS_PUBLIC]
         return amo.VALID_FILE_STATUSES
 
-    def find_latest_public_version(self):
-        """Retrieve the latest public version of an addon.
+    def find_latest_public_listed_version(self):
```

Comment:
My reasoning was, the `status` for the unlisted versions (well, files of the unlisted versions) is still going to be `STATUS_PUBLIC`, so the old name could still be confusing. And it's a method that's only supposed to be called internally anyway.

Still, I keep going back and forth on this. What do you prefer?